### PR TITLE
Feat/fee 355 extraction

### PR DESCRIPTION
**Pull request by maciejrybaniec**

### CODE REVIEW

File: src/components/ConfirmExtraction/ConfirmExtraction.tsx [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+  Footer,
+  Cancel,
+} from './ConfirmExtraction.styles';
+
+import {
+  getExtractionConfirmation,
+  queriesActions,
+} from '../../modules/queries';
+
+type Props = {};
```
Comment (apryka) [inline src/components/ConfirmExtraction/ConfirmExtraction.tsx:27]: not necessary if empty

Comment (maciejrybaniec) [reply src/components/ConfirmExtraction/ConfirmExtraction.tsx]: Fixed! Thanks :)

File: public/locales/en/translation.json [hunk 2]
```diff
@@ -123,6 +131,7 @@
     "new_query_title": "New Query"
   },
   "notifications": {
+    "execute_extraction_error": "We were not able to perform extraction. Please try again later."
```
Comment (apryka) [inline public/locales/en/translation.json:134]: missing comma at then end

File: src/modules/queries/constants.ts [hunk 3]
```diff
@@ -25,6 +26,9 @@ export const SET_QUERY_LIMIT_REACHED = '@queries/SET_QUERY_LIMIT_REACHED';
 export const GET_ORGANIZATION_USAGE_LIMITS =
   '@queries/GET_ORGANIZATION_USAGE_LIMITS';
 export const CLONE_SAVED_QUERY = '@queries/CLONE_SAVED_QUERY';
+export const SET_EXTRACTION_CONFIRMATION =
+  '@queries/SET_EXTRACTION_CONFIRMATION';
+export const CANCEL_EXTRACTION = '@queries/CANCEL_EXTRACTION';
```
Comment (mczapkowicz) [inline src/modules/queries/constants.ts:31]: Constant values ​​that are no longer associated with redux can be deleted

File: public/locales/en/translation.json (hunk 2 above)
Comment (maciejrybaniec) [reply public/locales/en/translation.json]: Fixed

File: src/modules/queries/constants.ts (hunk 3 above)
Comment (maciejrybaniec) [reply src/modules/queries/constants.ts]: Thanks! Fixed :)

**Review by apryka — APPROVED** [review]

**Pull request merged**